libs/core/src/lib/time/time.component.ts

```diff
@@ -88,15 +99,32 @@ export class TimeComponent implements OnChanges, ControlValueAccessor {
     @Input()
     time: TimeObject = { hour: 0, minute: 0, second: 0 };
 
+    /**
+     * @Input When set to false, hides the buttons that increment and decrement the corresponding columns.
+     */
+    @Input()
+    spinners: boolean = true;
+
     /** @hidden */
     @Output()
-    readonly focusArrowLeft: EventEmitter<void> = new EventEmitter<void>();
+    readonly focusFirstElement: EventEmitter<void> = new EventEmitter<void>();
 
     /** @hidden
      * Used only in meridian mode. Stores information the current am/pm state.
      */
     period: string;
 
+    /** @hidden container for [1 - 12/24] values */
+    hours: number[];
+
+    /** @hidden container for [1 - 60] values */
+    minutes: number[];
+
+    /** @hidden container for [am, pm] values */
+    periods: string[];
```

Comment:
I think you should keep naming this `meridians` to stay consistent.

Reply:
It was leftover, replaced by focus trap